lib/potluck.rb

```diff
@@ -0,0 +1,45 @@
+require './lib/dish'
+require 'pry'
+
+class PotLuck
+
+  attr_reader :date, :dishes
+
+  def initialize(date)
+    @date = date
+    @dishes = []
+  end
+
+  def add_dish
```

Comment:
This is sooo close!!  Pay attention to the spec - we need to pass an argument into `add_dish` and that argument should be a pre-existing Dish object.